aws_cf_templates/cz-pycon-2015-hosted-zone.template

```diff
@@ -0,0 +1,70 @@
+{
+    "AWSTemplateFormatVersion": "2010-09-09",
+    "Description": "Deploy hosted zone for cz-pycon-2015, v0.1 2015-06-28",
+
+    "Parameters": {
+        "DomainName": {
+            "Type": "String",
+            "Description": "The name of the domain. For resource record types that include a domain name, specify a fully qualified domain name",
+            "Default": "pycon.cz"
+        },
+        "DNSRecordType": {
+            "Type": "String",
+            "Description": "The type of record to add",
+            "Default": "A",
+            "AllowedValues": [
+                "A"
+            ]
+        },
+        "S3BucketHostedZoneId": {
+            "Type": "String",
+            "Description": "The hosted zone ID. For Amazon S3, use the hosted zone ID for your bucket's website endpoint",
+            "Default": "Z1BKCTXD74EZPE"
```

Comment:
This value is valid for all S3 buckets in the "eu-west-1" region. If you will deploy the template in another region, you must change this value to reflect region website endpoint. Please see http://docs.aws.amazon.com/general/latest/gr/rande.html#s3_region . For example if you will deploy a stack in the "us-east-1" region, right value is Z3AQBSTGFYJSTF .